### DISCUSSION TITLE
Scale searchlight (non-turret) rotation speed with Helm skill

### DISCUSSION THREAD

**Discussion by MatthiasKunnen** [FakeFishGames/Barotrauma#15093] [Ideas and suggestions]
In the same way that turret rotation speed scales with Weapon skill level, could the rotation speed of searchlights be scaled with Helm speed?

I have Weapon 79 and Helm 84 and it's incredible how much slower the search light moves compared to the turrets. It takes ages to move it 180°.

Perhaps searchlight range could also scale with Helm skill, though this would be a separate discussion.

**Comment by jigardi** [comment]
I agree with OP.
The disparity seems especially odd, when the objects appear so similar in function.
I wonder if a mod could emulate the desired functionality somehow.

**Reply by NoahRoopull** [reply]
I believe all that is needed is to give searchlights the "turret" tag, but that makes them scale with Weapon skill instead of Helm skill. Perhaps an attribute could be added to turrets to scale to the appropriate skill instead of always using Weapons.

**Reply by gwrzgrke** [reply]
Currently the searchlights too are turrets in that sense, that they do have low and high skill turning speed, which is related to weapons skill.
But on most ships, both low and high skill value are set to comically low speeds, most of the time less than the turret low skill value.
